src/main/java/github/weichware10/toolbox/gui/Tutorial.java

```diff
@@ -0,0 +1,104 @@
+package github.weichware10.toolbox.gui;
+
+import java.io.IOException;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+
+import github.weichware10.toolbox.codecharts.CodeCharts;
+import github.weichware10.toolbox.zoommaps.ZoomMaps;
+import github.weichware10.util.Files;
+import github.weichware10.util.Logger;
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wrong lexicographical order for 'github.weichware10.util.Logger' import. Should be before 'java.util.ArrayList'.